### ISSUE TITLE
Integrate this into openmm org?

### ISSUE THREAD

**Issue by jchodera** [rmcgibbo/openmm-cmd#35]
@rmcgibbo @peastman: If this is no longer being actively developed, what would you think about moving this to the http://github.com/openmm org, with the hope that we could eventually get developer resources to maintain it (or integrate it with OpenMM directly)? There is a good possibility we will be able to get funding for OpenMM in the near future.

Also, I note that there is an `openmm` PyPI package:
https://pypi.org/project/openmm/

Would it be OK to turn over the keys to that in case we figure out how to build a pip-installable OpenMM at some point?

**Comment by rmcgibbo** [comment]
Sorry for the delayed response (vacation).

Sure. I'm happy to transfer ownership of this repository to a different organization.

It's a little awkward that this is the "openmm" on pypi, rather than the actual OpenMM toolkit.

@peastman: I made you an owner of the "OpenMM" on pypi, and I think you're a "collaborator" on the github project here so you should have the authority to transfer the github repo to another organization and/or rename it. Feel free to do whatever you think is appropriate without my involvement.

**Comment by jchodera** [comment]
@rmcgibbo Thanks!

If we ever figure out how to build pypi wheels for OpenMM that include GPU support, we might consider replacing the `openmm` pypi packages with it, since this would allow us to deploy OpenMM easily in google colab and other online Jupyter notebooks to help with educational materials.

**Comment by jchodera** [comment]
(We could find a new home for `openmm-cmd` on pypi that better reflects the distinction between the two packages!)

**Comment by leeping** [comment]
I agree it would be great for the openmm org to take ownership of this and
for it to be maintained (perhaps becoming part of OpenMM) in the future!  :)

- Lee-Ping
